docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+
+version: '3.4'
+
+services:
+  database:
```

Comment:
Invalid syntax. One tab whitespace is required. And, you should rename service name with `s_mssqlserver` instead `database`.